AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/releasing/boundary/ResourceTypeEntityDto.java

```diff
@@ -0,0 +1,30 @@
+package ch.puzzle.itc.mobiliar.business.releasing.boundary;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public class ResourceTypeEntityDto implements Comparable<ResourceTypeEntityDto> {
+    @Getter
```

Comment:
Same as above. Annotate the class, not the properties (if all properties are annotated)